pkgs/top-level/python-packages.nix

```
@@ -6357,6 +6361,8 @@ in {
 
   pysmb = callPackage ../development/python-modules/pysmb { };
 
+  pySMART_smartx = callPackage ../development/python-modules/pySMART_smartx { };
```

```suggestion
  pysmart-smartx = callPackage ../development/python-modules/pysmart-smartx { };
```

Please also rename the pname.